firebase-perf/src/main/java/com/google/firebase/perf/FirebasePerformance.java

```diff
@@ -191,6 +193,14 @@ public static FirebasePerformance getInstance() {
     gaugeManager.setApplicationContext(appContext);
 
     mPerformanceCollectionForceEnabledState = configResolver.getIsPerformanceCollectionEnabled();
+    if (isPerformanceCollectionEnabled()) {
+      Log.i(
```

Comment:
Why don't you use the AndroidLogger instead of exposing the LogWrapper? It has the added advantage that it already does the check as well.

Reply:
The purpose is to log regardless of `isLogcatEnabled` defined in `AndroidLogger`. We want to log the console URL if FirebasePerformance is initialized per run.

Reply:
Thanks for the explanation. If `isLogcatEnabled = false`, wouldn't logging go against the user's expectation / desires?

Reply:
You are right. This decision is made based on the fact that not many developers will turn on the `isLogcatEnabled` and the flag is off by default. As a result, we would like to log only the dashboard URL only once per run. Happy to hear your thoughts.

Reply:
@jeremyjiang-dev How are we logging in the rest of the Firebase codebase? I would align with that. I would not recommend to supersede the choice of the developer to show/hide console logs.

Reply:
> I would not recommend to supersede the choice of the developer to show/hide console logs

+1

Reply:
Sg. I should have highlighted this in the design doc to catch attention.

Reply:
Done.